### PR TITLE
Small sounds update and misc stuff

### PR DESCRIPTION
**Pull request by Zth--**
:sound: Sounds added:
- Buckling and unbuckling to chairs
- Flashlights
- Flare sound too
- Taser
- Announcement
- Bodyfall (resisting fires)

Misc:
- Jiteriness when touching a powered wire
- Resisting fires is cool now
# :fire: **_WEBM LINK HERE_**: http://puu.sh/mX2gX/ca2d16c3c7.webm :fire:

^^^
webmm

### CODE REVIEW

**Comment by 9600bauds** [comment]
> for(var/i in list(1, 4, 2, 8, 1, 4, 2, 8, 1, 4))

Clusterfack is gonna have an aneurysm when he sees this

**Comment by Zth--** [comment]
It's copypasted from gyro tea code. What's a better way of doing it?

**Comment by 9600bauds** [comment]
I think Ian also does it, anyways it should probably at least be using defines.
So like for(var/i in list(NORTH, SOUTH, WEST etc
but then again who gives a shit

File: code/modules/mob/living/carbon/carbon.dm [hunk 1]
```diff
@@ -143,6 +143,8 @@
 
 	//if(src.stunned < shock_damage)	src.stunned = shock_damage
 
+	jitteriness += 20 //Shake that body, friend
```
Comment (clusterfack) [inline code/modules/mob/living/carbon/carbon.dm]: You use a proc to set this last I checked

File: code/modules/mob/living/living.dm [hunk 2]
```diff
@@ -917,9 +917,15 @@ Thanks.
 	//putting out a fire
 		if(CM.on_fire && CM.canmove)
 			CM.fire_stacks -= 5
-			CM.weakened = 5
+			CM.SetWeakened(3)
+			playsound(CM.loc, 'sound/effects/bodyfall.ogg', 50, 1)
 			CM.visible_message("<span class='danger'>[CM] rolls on the floor, trying to put themselves out!</span>",
 							   "<span class='warning'>You stop, drop, and roll!</span>")
+
+			for(var/i in list(1, 4, 2, 8, 1, 4, 2, 8, 1, 4))
```
Comment (clusterfack) [inline code/modules/mob/living/living.dm]: Shit son that is UGLY

Comment (9600bauds) [inline code/modules/mob/living/living.dm]: I warned you bro

Comment (clusterfack) [inline code/modules/mob/living/living.dm]: All this is is a loop that does dir = turn(dir, 90), you can literally replace it with that

Comment (9600bauds) [inline code/modules/mob/living/living.dm]: Wouldn't that do 1, 2, 4, 8?

Comment (clusterfack) [inline code/modules/mob/living/living.dm]: Last I checked 1 and 2 were north and south, and 4 and 8 were east and west

Comment (RemieRichards) [inline code/modules/mob/living/living.dm]: NORTH 1
SOUTH 2
EAST 4
WEST 8
@clusterfack is correct

Comment (9600bauds) [inline code/modules/mob/living/living.dm]: That's silly

Comment (RemieRichards) [inline code/modules/mob/living/living.dm]: @9600bauds it's so they can be used as bitflags to generate the diagonals
NORTH and EAST is 1 and 4, 5.
SOUTH and EAST is 2 and 4, 6.

and so on.
Directions being bits allows for a lot of common operations based on direction to be fast.
they also allow you to store multiple directions in one variable easily.

Comment (9600bauds) [inline code/modules/mob/living/living.dm]: Okay that makes more sense

Comment (Zth--) [inline code/modules/mob/living/living.dm]: Thanks, gonna try it

**Comment by Fox-McCloud** [comment]
I recommend TG's implementation of making someone flop around: https://github.com/tgstation/-tg-station/blob/8391ed1c23fc88f3ed034e96cd5f4378ad17a78c/code/modules/mob/living/carbon/carbon.dm#L400

 https://github.com/tgstation/-tg-station/blob/8391ed1c23fc88f3ed034e96cd5f4378ad17a78c/code/modules/mob/living/carbon/carbon.dm#L367

**Comment by clusterfack** [comment]
Wow it's literally the same with a switch, how awful, just use Turn people

File: code/modules/mob/living/living.dm (hunk 2 above)
Comment (Zth--) [inline code/modules/mob/living/living.dm]: Well I can't get it to work with that method. I'll ask at IRC when I get some free time

**Comment by ghost** [comment]
:+1: other than that awful code

**Comment by Zth--** [comment]
Rude

**Comment by Zth--** [comment]
Ok this one is ready to go.

**Comment by ihadtoregisterforthis** [comment]
I'm unsure about the buckling one. The message about the buckling when blind hints metal clanking.

Does the taser sound still play when the subject is not affected?

**Comment by Zth--** [comment]
Why wouldn't the taser sound play when the user is not affected? Think of a horrorform ling, the taser hits the ling, it's not like its missing, but he's inmune.

And I'm not sure what you're trying to say with the unbuckling one

**Comment by IratePirate** [comment]
More sounds are always good. Excellent work.

**Comment by ihadtoregisterforthis** [comment]
When as a living carbon you lose sight, a message plays whenever anyone on range buckles to a charity or bed, and it says "you hear metal clanking". This differs from the new sound.

**Comment by 9600bauds** [comment]
I haven't tested or listened to any of it so the best I can give you is NFD

**Comment by Zth--** [comment]
(There's a webm that shows literally everything at OP)

**Pull request merged**